content/en/agent/remote_config/_index.md

```diff
@@ -215,8 +220,16 @@ The following table describes the meaning of each Tracing library status:
   | NOT DETECTED   | The Tracing library does not support Remote Configuration. To fix this issue, update the Tracing library software to the latest available version. |
   | UNKNOWN   | The Tracing library status is unknown, and it can't be determined if an Agent is associated with the Tracing library. For example, this could be because the Agent is deployed on a fully managed serverless container service like AWS Fargate. |
 
-## Opting out of Remote Configuration at the Agent level
+## Opting out of Remote Configuration 
+To opt-out of Remote Configuration use, you can simply disable Remote Configuration at the organization level. Optionally, you can also disable Remote Configuration capability at the API key level and agent level.
 
+### At the Organization level
+Disable Remote Configuration at the organization level on the [Remote Configuration][8] page. This disables Datadog components across your organization to receive configurations from Datadog. You need the [`org_management`][7] permission to disable Remote Configuraiton at the organization level.
+
+### At the API Key level
+Disable the API key of your choice on the [API Keys][24] page. You need the [`api_keys_write`][5] permission to disable Remote Configuration capability on an API key.
+
+### At the Agent level
 Starting with Agent version 7.47.0, `remote_configuration.enabled` is set to `true` by default in the Agent. This setting causes the Agent to request configuration updates from the Datadog site.
 
 To receive configurations from Datadog, you also need to take the following steps:
```

Comment:
Is this section (this line, plus the following three bullet points) in the correct place? seems like it should be moved higher

Reply:
Yes, we believe its the right place as it follows the onboarding process which starts off with org-level, API key level, and Agent-level settings.